.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "brightspace/polymer-3-config"
+  "extends": "brightspace/wct-polymer-3-config"
```

Comment:
I'm not sure what this is so will leave it to one of our polymer 3 conversion gurus to approve this PR.

Reply:
So the converter was expecting `brightspace/wct-config` in the original, so messed up the conversion. Here's an example of how things should be in the hybrid:
https://github.com/BrightspaceUI/inputs/blob/hybrid/test/.eslintrc.json

I'd recommend making this fix in the hybrid version, then re-running the conversion. That way you won't need [this patch](https://github.com/Brightspace/polymer-3-converter/pull/95).